CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Optimizely Python SDK Changelog
 
+## 3.8.0
+February 11th, 2021
+
+### New Features
+* New Features
+Introducing a new primary interface for retrieving feature flag status, configuration and associated experiment decisions for users ([#309](https://github.com/optimizely/python-sdk/pull/309)). The new OptimizelyUserContext class is instantiated with createUserContext and exposes the following APIs to get OptimizelyDecision:
+
+    - setAttribute
+    - decide
+    - decideAll
+    - decideForKeys
+    - trackEvent
```

Comment:
Change all these api names to python ones (snake-case)

Reply:
done